kube-runtime/src/controller/mod.rs

```diff
@@ -18,7 +14,8 @@ use derivative::Derivative;
 use futures::{
     channel,
     future::{self, BoxFuture},
-    stream, Future, FutureExt, SinkExt, Stream, StreamExt, TryFuture, TryFutureExt, TryStream, TryStreamExt,
+    stream::{self, BoxStream},
```

Comment:
Unused import?

Reply:
This was just reordered by the module rename and restructuring, then reformatted by rustfmt. There were no unused imports here.

Reply:
Fair enough, misread the diff then.